modules/schematics/src/data/files/__name@dasherize@if-flat__/__name@dasherize__.ts.template

```diff
@@ -0,0 +1,3 @@
+export interface <%= classify(name) %> {
+  id?: any;
```

Comment:
Is having an `id` not a requirement?

```suggestion
  id: any;
```

Reply:
It is just to suggest the interface to the developer. Setting it as required property on creating a new object in CRUD we do not have the id before receiving from backend it. But it is just a suggestion.